cmd/sys-mgmt/res/configuration-docker.toml

```diff
@@ -0,0 +1,50 @@
+ConsulProfilesActive = 'docker;go'
+ReadMaxLimit = 100
+MetaDataCheck = false
+ValidateCheck = false
+AddToEventQueue = true
+PersistData = true
+AppOpenMsg = 'This is the Core Data Micro Service'
+FormatSpecifier = '%(\\d+\\$)?([-#+ 0(\\<]*)?(\\d+)?(\\.\\d+)?([tT])?([a-zA-Z%])'
+MsgPubType = 'zero'
+ServicePort = 48080
+ServiceTimeout = 5000
+ServiceAddress = 'edgex-core-data'
+DeviceUpdateLastConnected = false
+ServiceUpdateLastConnected = false
+DBType = 'mongodb'
+MongoDBUserName = 'core'
+MongoDBPassword = 'password'
+MongoDatabaseName = 'coredata'
+MongoDBHost = 'edgex-mongo'
+MongoDBPort = 27017
+MongoDBConnectTimeout = 5000
+ConsulHost = 'edgex-core-consul'
+ConsulCheckAddress = 'http://edgex-core-data:48080/api/v1/ping'
```

Comment:
This is all just cut-and-pasted as is the non-docker configuration file. Please review these with an eye toward what your service actually needs. If it doesn't need anything right now, then remove all this stuff.